src/pages/apps/index.tsx

```diff
@@ -40,6 +51,7 @@ const Apps: NextPage = () => {
     remoteSafeAppsLoading,
   })
 
+  // TODO: add a loading state
```

Comment:
does this need design? if yes i'd create a github issue instead so we're less likely to forget

Reply:
I removed the comment!